### ISSUE TITLE
frictionless validate  --schema: if I forget to enter an input file, the shell waits for a response

### ISSUE THREAD

**Issue by aborruso** [frictionlessdata/frictionless-py#835]
# Overview

Hi,
I have this schema

```
fields:
  - name: denominazione_comune
    type: string
  - name: lat
    type: number
  - name: long
    type: number
```

If I run `frictionless validate  --schema input.yaml` without inserting the file to validate, nothing happens.

Thank you

---

Please preserve this line to notify @roll (lead of this repository)

**Comment by roll** [comment]
Thanks!

I'm releasing a fix with `frictionless@4.9.3`

**Comment by aborruso** [comment]
Thank you very much @roll
